src/safeds_stubgen/docstring_parsing/_docstring.py

```diff
@@ -47,10 +48,13 @@ def to_dict(self) -> dict[str, Any]:
         return dataclasses.asdict(self)
 
 
+@dataclass(frozen=True)
+class ResultDocstrings:
+    docstrings: Sequence[ResultDocstring]
+
+
```

Comment:
What is the purpose of this class, as opposed to passing a list of `ResultDocstring` around?

Reply:
That's... a good question. I removed it.